packages/components/layout/src/Layout.styles.ts

```diff
@@ -20,28 +30,107 @@ export const getLayoutMaxWidthStyles = ({
     boxShadow: withBoxShadow
       ? '0px 6px 16px -2px rgba(25, 37, 50, 0.1), 0px 3px 6px -3px rgba(25, 37, 50, 0.15)'
       : 'unset',
-    borderRadius: `10px 10px 0 0`,
-    margin: `0 ${tokens.spacingM}`,
+    borderRadius: '10px 10px 0 0',
+    margin: `0 auto`,
+    width: `calc(100% - ${tokens.spacingM} * 2)`,
+  });
+};
+
+const getContentContainerGridTemplateColumns = ({
+  variant,
+  withLeftSidebar,
+  withRightSidebar,
+}: {
+  variant: LayoutContextType['variant'];
```

Comment:
Could we have used the already imported LayoutProps instead?

```suggestion
  variant: LayoutProps['variant'];
```